pages/Add/index.js

```diff
@@ -27,16 +16,11 @@ export default function Add() {
       fishweight: +e.target.weight.value,
```

Comment:
Everything looks good! Just a small suggestion: use camelCase or snake_case in naming the object keys. 😁

Reply:
ah, true that one! Thanks :)